pkg/watcher/reconciler/pipelinerun/reconciler.go

```diff
@@ -17,32 +17,31 @@ package pipelinerun
 import (
 	"context"
 	"fmt"
+	"time"
 
 	"github.com/tektoncd/results/pkg/apis/config"
 	"github.com/tektoncd/results/pkg/pipelinerunmetrics"
 
 	pipelinev1 "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1"
 	"github.com/tektoncd/pipeline/pkg/client/clientset/versioned"
+	pipelinerunreconciler "github.com/tektoncd/pipeline/pkg/client/injection/reconciler/pipeline/v1/pipelinerun"
 	pipelinev1listers "github.com/tektoncd/pipeline/pkg/client/listers/pipeline/v1"
 	"github.com/tektoncd/results/pkg/watcher/reconciler"
+
 	resultsannotation "github.com/tektoncd/results/pkg/watcher/reconciler/annotation"
 	"github.com/tektoncd/results/pkg/watcher/reconciler/dynamic"
 	"github.com/tektoncd/results/pkg/watcher/results"
 	pb "github.com/tektoncd/results/proto/v1alpha2/results_go_proto"
 	"go.uber.org/zap"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
-	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/client-go/kubernetes"
-	"k8s.io/client-go/tools/cache"
 	"knative.dev/pkg/controller"
 	"knative.dev/pkg/logging"
 	knativereconciler "knative.dev/pkg/reconciler"
 )
 
 // Reconciler represents pipelineRun watcher logic
 type Reconciler struct {
-	// Inline LeaderAwareFuncs to support leader election.
-	knativereconciler.LeaderAwareFuncs
```

Comment:
What's the effect of removing leader awareness? what happens if user is running multiple instances of the watcher? Shouldn't we document the effects of that change in the commit message/release notes?